packages/core/vue/src/useWallet.ts

```diff
@@ -38,13 +37,13 @@ export interface WalletStore {
         options?: SendTransactionOptions
     ): Promise<TransactionSignature>;
 
-    signTransaction: Ref<SignerWalletAdapter['signTransaction'] | undefined>;
-    signAllTransactions: Ref<SignerWalletAdapter['signAllTransactions'] | undefined>;
-    signMessage: Ref<MessageSignerWalletAdapter['signMessage'] | undefined>;
+    signTransaction: Ref<SignerWalletAdapterProps['signTransaction'] | undefined>;
+    signAllTransactions: Ref<SignerWalletAdapterProps['signAllTransactions'] | undefined>;
+    signMessage: Ref<MessageSignerWalletAdapterProps['signMessage'] | undefined>;
 }
 
 export interface WalletStoreProps {
-    wallets: Wallet[];
+    wallets: Adapter[];
```

Comment:
Probably what makes sense to do is expose this, or at least alias it internally, as `adapters` and instances are `adapter`. Since this is a breaking change anyway, let's nail down the distinction.

Reply:
I tried to keep the interfaces the same but made some changes to the internal methods to keep the use of 'adapters' and 'adapter' consistent